test/unit/util/define-dynamic-block.test.js

```diff
@@ -0,0 +1,202 @@
+import defineDynamicBlock from '../../../src/lib/define-dynamic-block';
+
+import BlockType from 'scratch-vm/src/extension-support/block-type';
+
+const MockScratchBlocks = {
+    OUTPUT_SHAPE_HEXAGONAL: 1,
+    OUTPUT_SHAPE_ROUND: 2,
+    OUTPUT_SHAPE_SQUARE: 3
+};
+
+const categoryInfo = {
+    name: 'test category',
+    color1: '#111',
+    color2: '#222',
+    color3: '#333'
+};
+
+const penIconURI = 'data:image/svg+xml;base64,fake_pen_icon_svg_base64_data';
+
+const testBlockInfo = {
```

Comment:
hmmm... probably we should have a test block with arguments...?

Reply:
Ah I see what we're doing below. I guess this is a todo along w/ the todo for `interpolate_`

Reply:
LLK/scratch-gui#4929